01_02_Combat_Rules.txt

```diff
@@ -186,6 +186,26 @@ and becomes Unconscious (and does not gain the Bleeding Out condition).
 	A target reduced to exactly 1 health by a nonlethal attack does not 
 fall Unconscious. 1 is not 0.
 
+Melee Accuracy and Guard Critical Hits:
+	See "Critical Hits" below for general Critical information. As normal, 
+a Critical Accuracy roll gives its attack +20 damage, or +30 on an Extra-critical. 
```

Comment:
nonblocker: a little concerned about creating duplicate points in our rules that say the same thing; if we tune crits up or down, we'll need to edit this line too.

Reply:
A good point, I can be less specific. "Melee Criticals deal extra damage as normal." or so.